modules/client/src/multisig/interfaces.ts

```diff
@@ -42,15 +42,15 @@ export interface IMultisigClientMethods extends IClientCore {
 }
 
 export interface IMultisigClientEncoding extends IClientCore {
-  addAddressesAction: (params: AddAddressesParams) => DaoAction;
-  removeAddressesAction: (params: RemoveAddressesParams) => DaoAction;
+  addAddressesAction: (params: AddAddressesParams) => DaoAction[];
+  removeAddressesAction: (params: RemoveAddressesParams) => DaoAction[];
   updateMultisigVotingSettings: (
     params: UpdateMultisigVotingSettingsParams,
   ) => DaoAction;
 }
 export interface IMultisigClientDecoding extends IClientCore {
-  addAddressesAction: (data: Uint8Array) => string[];
-  removeAddressesAction: (data: Uint8Array) => string[];
+  addAddressesAction: (data: Uint8Array[]) => MultisigPluginSettings;
+  removeAddressesAction: (data: Uint8Array[]) => MultisigPluginSettings;
```

Comment:
This is going to be tricky. Unless we receive exactly two actions, things will fail? How is people going to detect this?

I would center on the encoding part, and leave the decoding side as a simple piece